### PR TITLE
epi v2 finalization

### PR DESCRIPTION
**Pull request by munnsmunns**
Implements requested changes from the most recent Epi group feedback

- Change "Create" button to "Add Row"
- Add Hair to biomonitoring choices
- Add "Units" field to Data extraction
- Make Source, Age category, Exposure measurement types required
- Make Sex, Type of variance estimate (assuming variance is not filled out), Lower/upper interval type (assuming lower/upper interval is not filled out), Lower/upper bound type (assuming lower/upper bound is not filled out) not required
- double length of AdjustmentFactor description field
- Add a symbols dropdown to forms for easy copy/pasting of greek symbols, super/subscripts, etc.
- Re-order Outcome fields and rename to System, Effect, Effect detail, and Endpoint/outcome
- Central tendency displays median or mean if available, does not show range in the table view

Remove the feature flag, clean up migrations, and release!

### CODE REVIEW

File: hawc/apps/epiv2/constants.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ class VarianceType(models.IntegerChoices):
 
 
 class ConfidenceIntervalType(models.TextChoices):
+    NONE = "NA", NO_LABEL
```
Comment (munnsmunns) [inline hawc/apps/epiv2/constants.py:66]: There are a few fields like this where the scientists asked us to make them not required, but they already have an option that is "---" or "Not reported". Maybe the "---" is confusing and they think it isn't a valid option? or we should just hide the asterisk on those fields?

File: hawc/apps/epiv2/constants.py [hunk 2]
```diff
@@ -50,6 +50,7 @@ class EffectEstimateType(models.TextChoices):
     SCC = "SCC", "Spearman's Correlation Coefficient"
     PC = "PC", "Percent change"
     MD = "MD", "Mean difference"
```
[the remark this answers is not in the document]
Comment (munnsmunns) [reply hawc/apps/epiv2/constants.py:52]: For all fields with choices or just effect estimate type?

[the remark this answers is not in the document]
Comment (shapiromatron) [reply hawc/apps/epiv2/constants.py:52]: Just this one, b/c now if you type in "other" you'll specify a full value in the table

File: hawc/apps/epiv2/constants.py (hunk 1 above)
Comment (munnsmunns) [reply hawc/apps/epiv2/constants.py:70]: Perhaps N/A would be a better display value for these fields?

Comment (shapiromatron) [reply hawc/apps/epiv2/constants.py:70]: agree!

**Review by shapiromatron — CHANGES_REQUESTED** [review]
Since this will be our release, can we also tear out the "coming soon" stuff that we previously had:

1. Remove the ENABLE_EPI_V2 feature flag
2. Update the `Study detail` page to allow for epi v2 extractions

**Review by shapiromatron — APPROVED** [review]
Nice job @munnsmunns and @casey1173! Excited to release this!

**Pull request merged**